locales/en/moves.json

```diff
@@ -133,6 +133,9 @@
     "no_results": "No moves requested",
     "single_moves": "Single moves",
     "single_moves_no_results": "No moves proposed",
+    "created_at": "Created at",
+    "move_type": "Move type",
+    "date": "Date",
```

Comment:
Maybe `date` should live in the default file? Feels like something that has more generic use than being tied to the moves dashboard.